src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -28,5 +28,8 @@ public WatermarkOffsets(Offset low, Offset high)
 
         public Offset Low { get; }
         public Offset High { get; }
+
+        public override string ToString()
+            => $"[{Low}, {High}]";
```

Comment:
Seems like anything to indicate a range would be fine here -- otherwise this patch LGTM